package.json

```diff
@@ -17,9 +19,12 @@
   },
   "dependencies": {
     "aws-sdk": "^2.218.1",
+    "casperjs": "^1.1.4",
```

Comment:
Actually I was trying to hack Weibo using `casperjs`, `nitghwatch` or `phantomjs` without success. I will remove those useless dependencies for now.